torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -1335,19 +1335,64 @@ const std::vector<std::string> functions = {
         def clamp(self,
                   min: Optional[number],
                   max: Optional[number]):
-          def backward(grad_output):
-            if min is not None and max is not None:
-                mask = ((self >= float(min)) * (self <= float(max))).type_as(self)
-                return grad_output * mask, None, None
-            elif min is not None:
-                mask = (self >= float(min)).type_as(self)
-                return grad_output * mask, None, None
-            elif max is not None:
-                mask = (self <= float(max)).type_as(self)
-                return grad_output * mask, None, None
-            else: #min is None and max is None
-                return grad_output, None, None
-          return torch.clamp(self, min=min, max=max), backward
+            def backward(grad_output):
+                if min is not None and max is not None:
+                    mask = ((self >= float(min)) * (self <= float(max))).type_as(self)
+                    return grad_output * mask, None, None
+                elif min is not None:
+                    mask = (self >= float(min)).type_as(self)
+                    return grad_output * mask, None, None
+                elif max is not None:
+                    mask = (self <= float(max)).type_as(self)
+                    return grad_output * mask, None, None
+                else: #min is None and max is None
+                    return grad_output, None, None
+            return torch.clamp(self, min=min, max=max), backward
+
+        def clamp_with_tensors_0(self,
```

Comment:
What would happen if we didn't modify this file?

Reply:
Then the `TestJitGeneratedAutograd` tests for `clamp_with_tensors` would fail, since the TorchScript and PyTorch implementation gradients must match. The tests are generated for the definitions I've added in `common_methods_invocations.py` in `method_tests`. 

As far as I know, the `symbolic_script.cpp` definitions are required for all Aten functions.